src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWallet.scala

```diff
@@ -26,7 +26,7 @@ class ErgoWallet(historyReader: ErgoHistoryReader, settings: ErgoSettings)
   // Now these settings are hard-coded, however, they should be parameterized
   // https://github.com/ergoplatform/ergo/issues/856
   val maxInputs = 48
-  val optimalInputs = 6
+  val optimalInputs = 2
```

Comment:
please leave the precious experience comments in the code, why 6 should be changed to 2.

Reply:
No strict reason, just some common sense